modular_skyrat/modules/moretraitoritems/code/infiltrator_balaclava.dm

```diff
@@ -0,0 +1,31 @@
+/obj/item/clothing/mask/infiltrator
+	name = "infiltrator balaclava"
+	desc = "It makes you feel safe in your anonymity, but for a stealth outfit you sure do look obvious that you're up to no good. It seems to have a built in heads-up display."
+	icon_state = "syndicate_balaclava"
+	inhand_icon_state = "syndicate_balaclava"
+	clothing_flags = MASKINTERNALS
+	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDESNOUT
+	visor_flags_inv = HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
+	w_class = WEIGHT_CLASS_SMALL
+	armor = list(MELEE = 10, BULLET = 5, LASER = 5,ENERGY = 5, BOMB = 0, BIO = 50, FIRE = 100, ACID = 40)
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+	var/voice_unknown = FALSE ///This makes it so that your name shows up as unknown when wearing the mask.
```

Comment:
You're going to need to do more than this. This bit of code no longer does anything. You'll have to go back into ```human_say``` to fix it, as I removed the check for this mask's var.

Reply:
I'm aware, my first concern with mirrors is getting them to pass checks, then i go and fix the functions after